from swype.exceptions.unexpected_error import UnexpectedError


class ConfigurationError(UnexpectedError):
    pass
